import_streets/src/extract.rs

```diff
@@ -159,77 +146,7 @@ impl OsmExtract {
             return false;
         }
 
-        // Not sure what this means, found in Seoul.
-        if tags.is("lanes", "0") {
```

Comment:
I think I added this to avoid some crash. Just going to remove it as an old hack and if we hit whatever this was again, we can deal with it more properly hopefully